report/src/system_evaluation/results.tex

```diff
@@ -1,2 +1,70 @@
-\section{Results}\label{sec:experiment_results}
-%describe in terms of criteria for using bluetooth. `'
\ No newline at end of file
+\section{System evaluation}\label{sec:experiment_results}
+%describe in terms of criteria for using bluetooth. `'
+Having created a map that can be used to classify received measurements, we can now begin to examine wether these classifications are accurate. 
+The measurements are collected using the implemented functionality described in Section \ref{sec:knn_implementation}.
+However, while the room was unoccupied during the creation of the map, this was not the case for this part of the experiment. 
+Computers and phones where placed near the tables of the room, with several of them having either only WiFi or both Bluetooth and WiFi enabled. 
+In the common area outside the room, two people were tasked with moving around, and a laptop with Bluetooth and WiFi enabled was placed on one of the tables.
```

Comment:
Why are we saying two people were tasked with this? Is this relevant? If so we should also specify how many people did measurements inside the room.

Reply:
Aebeloe requested a really detailed explanation of how the experiment is performed. This is due to the fluctuations caused by even small changes to the physical environment.